README.md

```diff
@@ -65,6 +65,8 @@ Usage: data-validator [options]
   --help                   Show this help message and exit.
 ```
 
+If you want to build with a modern version of java set the "MODERN_JAVA" enviornment variable.
```

Comment:
**suggestion:** 
```suggestion
If you want to build with a modern version of Java, set the "MODERN_JAVA" environment variable.
```

Also, @holdenk, can you say what constitutes a modern version? I reckon it'd be 17+?

Reply:
Based on some searching around, the options added below appear to be available in JDK11+, so I think "modern" means JDK 11+… so I'm going to make it say that for now.